packages/ui/TimezoneChangeDialog.tsx

```diff
@@ -53,6 +54,10 @@ export default function TimezoneChangeDialog() {
     toast && showToast(t("we_wont_show_again"), "success");
   }
 
+  const { data } = useSession();
+
+  if (data?.user.impersonatedByUID) return null;
```

Comment:
copied and negated from https://github.com/calcom/cal.com/blob/464343f5abb1dbc95560498ca7c7a9bfe5a32480/packages/features/ee/impersonation/components/ImpersonatingBanner.tsx